minicluster/src/test/java/alluxio/multi/process/MultiProcessClusterTest.java

```diff
@@ -60,7 +60,8 @@ public void zookeeper() throws Exception {
   private void clusterVerification() throws Exception {
     try {
       mCluster.start();
-      mCluster.waitForAllNodesRegistered(60 * Constants.SECOND_MS);
+      // Give enough time for the cluster startup. See {@link MultiProcessClusterTest#mTimeout}
+      mCluster.waitForAllNodesRegistered(250 * Constants.SECOND_MS);
```

Comment:
250s seems a bit long to me. Which tests require a longer startup time than 60s?

Reply:
```
public final class MultiProcessClusterTest {
  private MultiProcessCluster mCluster;

  @Rule
  public Timeout mTimeout = Timeout.millis(Constants.MAX_TEST_DURATION_MS);

```

The MAX time for this test is 10min, and there are **two** test cases in this UT.
I attached a screen shot for this test running on my development machine.
The whole time to run this test is little more than 4 minutes. Each of them needs a little more than 2 minutes.

So, waiting 250 seconds for each of the sub test case, (and 500 seconds in total) is a good duration which will not exceed the total time out value.

Reply:
This is the cluster startup time that we're talking about, not the total running time for a test case. If something went wrong during startup, the test should abort ASAP, so that the developer can get feedback sooner.

Reply:
The `Timeout` rule applies to each one of the test methods, not the total time of the whole test class. So 10 minutes for each test case is adequate.

Reply:
>This is the cluster startup time that we're talking about, not the total running time for a test case. If something went wrong during startup, the test should abort ASAP, so that the developer can get feedback sooner.

But we don't know there is something wrong during startup. So we need to wait for some time to check the result.

In my local testing it sometimes take more than 60 seconds.
Approximately, 50% tests of cluster startup takes more than 60 seconds.

Reply:
Hold on. 
I did the test again, and it only took 10~12 seconds for each test case.
Let me check again.